mungegithub/mungers/submit-queue.go

```diff
@@ -55,7 +55,7 @@ const (
 	lgtmLabel                      = "lgtm"
 	retestNotRequiredLabel         = "retest-not-required"
 	retestNotRequiredDocsOnlyLabel = "retest-not-required-docs-only"
-	doNotMergeLabel                = "do-not-merge"
+	doNotMergePrefix               = "do-not-merge"
```

Comment:
If we wind up searching explicitly for `do-not-merge/foo` and `do-not-merge/bar` below how do you feel about eliminating this prefix? IMO it is easier for developers to find code if they can search for `do-not-merge/foo` literals in the code base.

IMO helping people search and understand other's code is more valuable here than the risk that the next plugin might fat-finger their spelling of `do-not-megre/spam`

Reply:
SGTM